draft/overlay/30a-relay-hat.md

```diff
@@ -0,0 +1,64 @@
+<!--
+---
+name: 30A Relay HAT
+class: board
+type: io, relay
+formfactor: HAT
+manufacturer: PlasmaDan
+description: Raspberry Pi HAT I/O board which offers an impressive 30A switching capability.
+url: https://plasmadan.com/30arelayhat
+github: https://github.com/plasmadancom/30A-Relay-HAT
+buy: https://plasmadan.com/30arelayhat
+image: '30a-relay-hat.png'
+pincount: 40
+eeprom: yes
+power:
+  '1':
+  '2':
+  '4':
+ground:
+  '6':
+  '9':
+  '14':
+  '20':
+  '25':
+  '30':
+  '34':
+  '39':
+pin:
+  '3':
+    mode: i2c
+  '5':
+    mode: i2c
+i2c:
+  '0x20':
+    alternate: [ '0x21', '0x22', '0x23', '0x24', '0x25', '0x26', '0x27' ]
+    name: MCP23008
+    device: MCP23008
+-->
+# 30A Relay HAT
+
+A Raspberry Pi HAT I/O board which offers an impressive 30A switching capability. Ideally suited to automation or industrial control, switching of household appliances, industrial machinery or automotive applications.
```

Comment:
Is "impressive" a bit subjective to be used on the Pinout site? :shrug:

Reply:
Yes, you're right. I've updated the description.